### PR TITLE
Updating docs for sample apps option in init

### PR DESCRIPTION
**Pull request by nikhname**
*Issue #, if available:*

*Description of changes:*
Added samples workflow and more information in the usage section.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: cli-toolchain/quickstart.md [hunk 1]
```diff
@@ -49,6 +49,12 @@ The `init` command goes through the following steps:
 - Insert amplify folder structure into the project's root directory, with the initial project configuration
 - Generate the project metadata files, with the outputs of the above-selected plugin(s)
 
+### Samples workflow
+To set up a sample amplify project, execute the following command inside an empty directory:
+
+`amplify init --app <github url>`
+
+where the github url is a valid sample amplify project repository. Click [here](https://aws-amplify.github.io/docs/cli-toolchain/usage#--app) for more details.
```
Comment (nikhil-dabhade) [inline cli-toolchain/quickstart.md]: We should add details on what we mean by a "valid" amplify project repo. 
For e.g. what are expecting in this project that makes it a valid project repo.

File: cli-toolchain/usage.md [hunk 2]
```diff
@@ -83,6 +84,20 @@ The aws secret access key if `useProfile` is set to false.
 - `region`: 
 The aws region if `useProfile` is set to false.
 
+##### `--app`
+Installs, initializes and provisions resources for a sample amplify application from the provided github repository url. This option must be executed in an empty directory. The sample repository must have an amplify folder, including the following:
```
Comment (nikhil-dabhade) [inline cli-toolchain/usage.md]: Installs, initializes and provisions resources for a sample amplify application from the provided github repository url. -->
Installs, initializes, and provisions resources for a sample amplify application from the provided GitHub repository URL.

File: cli-toolchain/quickstart.md (hunk 1 above)
Comment (nikhil-dabhade) [reply cli-toolchain/quickstart.md]: Seems like you have these details in the section that explains the "--app" option. 
Let's link from here to that option

File: cli-toolchain/usage.md (hunk 2 above)
Comment (nikhname) [reply cli-toolchain/usage.md]: Made this change

File: cli-toolchain/quickstart.md (hunk 1 above)
Comment (nikhname) [reply cli-toolchain/quickstart.md]: The 'Click [here]' links to the --app option

**Pull request merged**